File: ASP.NET/test/mvc/TelerikMvcApp/Models/Staff.cs
﻿using System;

namespace TelerikMvcApp.Models
{
    public class Staff
    {
        public long ID { get; set; }
        public string Name { get; set; }
        public decimal? Salary { get; set; }
        public int? Dep { get; set; }
        public DateTime? BirthDate { get; set; }
    }
}

File: ASP.NET/test/Sencha/ExtJS/ExtJS4/TestGrid/app/model/Staff.js
Ext.define("TG.model.Staff", {
    extend: "Ext.data.Model",
    idProperty: "Id",
    fields: [
        { name: "Id", type: "int" },
        "Name",
        { name: "Salary", type: "float" },
        { name: "Dep", type: "int" },
        { name: "BirthDate", type: "date", dateFormat: "MS" }
    ]
});
